### DISCUSSION TITLE
DXVK: "DxvkInstance: Failed to create instance"

### DISCUSSION THREAD

**Discussion by Random-Scientist** [The-Wineskin-Project/WineskinServer#330] [Q&A] [answered]
I'm getting an error with dxvk-macos on the latest wrapper / CXwine version. It seems not to be recognizing the libMoltenVK in Frameworks?
the error is 
`info:  Required Vulkan extension VK_KHR_surface not supported`

Is there any known fix for this? 
Previously, I fixed it by putting libMoltenVK.dylib in the wine/lib64 folder, but the library folders got reorganized in the newer wrapper version

Note that this is a similar behavior to #191, seems like there is something odd going on with libMoltenVK

**Comment by Gcenx** [comment, marked as answer]
Not sure _why_ your replacing MoltenVK but your assumption that MoltenVK isn’t being found is wrong.

> info: Required Vulkan extension VK_KHR_surface not supported

If libMoltenVK wasn’t found you’ll have seen a message saying it can’t be found not this message that’s complaining about an unsupported Vulkan extension.

Are you installing an old copy of MoltenVK?

**Reply by Random-Scientist** [reply]
Nope. This occurs on a *totally fresh* CX22 wrapper with untouched libMoltenVK, and I was replacing it the the latest version from your fork (the-wineskin-project/moltenvk). Downgrading to CX21 and putting it in sharedsupport/wine/lib64 made everything work fine again

**Reply by Gcenx** [reply]
This seems very odd.

In the current wrapper `sharedsupport/wine/lib64` takes priority over any other library location, I do allow Wineskin to use other external sources for libraries, do you happen to have another copy of libMoltenVK installed somewhere else in your system?

**Reply by Random-Scientist** [reply]
That’s most likely the culprit, I’ll check when I get home, but it’s probably an old moltenvk version installed system wide taking precedence

**Reply by Gcenx** [reply]
For the next wrapper update I’ll ensure Wineskin paths have a higher priority than system locations, `sharedsupport/wine/lib` & `sharedsupport/wine/lib64` will still take priority over /Frameworks.

**Reply by Random-Scientist** [reply]
That was it! Thanks again!
